docs/_templates/layout.html

```diff
@@ -4,3 +4,12 @@
     {{ super() }}
 	<a href="{{ pathto('genindex') }}">Keyword Index</a>
   {% endblock %}
+
+  {%- block extrahead %}
+	<script src="https://unpkg.com/wikipedia-preview@1.4.0/dist/wikipedia-preview.production.js"></script>
+	  <script>
+            wikipediaPreview.init({
+              detectLinks: true
+            })
+	  </script>
```

Comment:
The `<script>` has no `async` or `defer` attributes so if a request to unpkg.com is slow for any reason (can be just bad connection), it will block loading of any scripts that come after it. I think this should have `async` attribute and the `init()` call should be in some callback that fires when the script is already loaded.